WebService.Test/helpers/SomeException.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+
+namespace Services.Test.helpers
+{
+    public class SomeException : Exception
```

Comment:
what is the point of SomeException?  Are we going to implement it? At the least we should rename it.

Reply:
removed

Reply:
please restore this, it's used on purpose to avoid false positives. See my other comment. Please add a comment to the class explaining its purpose